13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+In these cases, wherein additional details may not be helpful to beginners, it can be valuable to important these questions (and the issues that are assocaited with them), but to have responses or answers that provide more clarity, rather than confusion. 
+
+For example, 
+
+<!-- Need to answer some common-sense ideas for how to respond to these? -->
+
+### Start lessons or activities with visualizing data
+
+There are examples from data science books (Wickham & Grolemund, 2017) and past research (e.g., Lehrer, Kim, & Schauble, 2007; see Lehrer & Schauble, 2015, for more examples) that suggests that starting with visualizing data can be benefician in terms of learners' ability to work with data. Wickham and Grolemund (2017) write that they begin their book, *Data Science Using R*, with a chapter on visualization, because doing so allows learners to create something that they can share immediately, whereas tasks such as loading data can be rife with issues - and does not immediately lead learners to have a product they can share. Lehrer, Kim, and Schauble (2007) show how providing students with an opportunity to invent statistics by displaying the data in new ways. This led to (productive) critique among fifth- and sixth-grade students and their teacher.
+
+### Consider representation in the data and examples you use
+
+One way to think about data is that it is objective, free of decisions about what to value or prioritize. Another is to consider data as a process that is value-laden, from deciding what question to ask (and what data to collect) to interpreting findings with attention to how others will make sense of them (e.g., O'Neill's *Weapons of Math Destruction*, and Lehrer, Kim, and Schauble's (2007) description of data modeling). From this broader view, choosing representative data is a choice, like others, that teachers can make. 
+
+We think this consideration is important - particularly when it comes to data-related issues that we consider to be objective, such as how variables are assumed and constructed to be dichotomous (such as variables for individual's gender) or categorical (such as variables for individual's race), when the truth may be that such variables are based on decisions that analysts or those collecting data made - decisions that may benefit from questioning. This consideration is also important when it comes to what data is used for teaching and learning. If names of individual's in data exclusively from individuals from majority racial or ethnic group, for example, some learners may implicitly perceive the content being taught to be designed for others. While we may think that such issues are better left to those we are teaching to decide later on, setting the stage in classes, courses, and other contexts in which data science is taught and learned can set an important precedent for the data our learners use and how they use it.
+
+### Provide a home base for learners to access resources (and to learn more)
```

Comment:
this is fantastic - and maybe should lead the section? we should also be sure that we're providing a "home base" in our text.

Reply:
move to start. great/cool idea about a home base in our text, adding to notes